frontend/src/configs/rdev.js

```diff
@@ -1,8 +1,7 @@
-const { PLAUSIBLE_DATA_DOMAIN_STAGING } = require("./common");
-
 const configs = {
   API_URL: "$API_URL",
-  PLAUSIBLE_DATA_DOMAIN: PLAUSIBLE_DATA_DOMAIN_STAGING,
+  // plausible doesn't expect  a protocol in the domain
+  PLAUSIBLE_DATA_DOMAIN: "$FRONTEND_URL".replace(/^https?:\/\//, ""),
```

Comment:
For production applications, this approach feels the most DRY and less of a headache should anything change upstream